stdlib/2and3/logging/__init__.pyi

```diff
@@ -160,8 +160,8 @@ class Logger(Filterer):
             exc_info: Optional[_ExcInfoType] = ...,
             extra: Optional[Dict[str, Any]] = ...,
         ) -> None: ...  # undocumented
-    def addFilter(self, filt: _FilterType) -> None: ...
-    def removeFilter(self, filt: _FilterType) -> None: ...
+    def addFilter(self, filter: _FilterType) -> None: ...
```

Comment:
`Logger` actually doesn't define this method, it's inherited from `Filterer`. From looking at the code, it seems like `Filterer.addFilter` should just take `_FilterType` (not `Filterer` as currently annotated). Could you make that change and remove the `addFilter` and `removeFilter` methods from Logger and Handler?